cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceusageevents/PurgeAndReseedServiceUsageEventsRequest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v2.serviceusageevents;
+
+import lombok.Builder;
+import lombok.Data;
+import org.cloudfoundry.client.Validatable;
+import org.cloudfoundry.client.ValidationResult;
+
+/**
+ * The request payload for the Purge and Reseed Service Usage Events operation
+ */
+@Data
+public class PurgeAndReseedServiceUsageEventsRequest implements Validatable {
```

Comment:
This needs a test like `GetInfoRequestTest`.  I'll add it in a following commit.